src/services/book.service.ts

```diff
@@ -7,3 +7,11 @@ export const getNumBooks = async () => {
     const numBooks = await Promise.all([bookRepository.count()]);
     return numBooks
 };
+
+export const getListBooks = async () => {
+    const listBooks = await bookRepository.find({
+        order: { title: 'ASC'},
+        relations: ['author']
+    })
+    return listBooks;
```

Comment:
fix tương tự file bên dưới e nha
```suggestion
    return await bookRepository.find({
        order: { title: 'ASC'},
        relations: ['author']
    });
```